package-inspector/src/main/java/com/microsoft/inspector/MainActivity.java

```diff
@@ -148,15 +150,22 @@ public void onItemClick(AdapterView<?> adapterView, View view, int position, lon
                         packageSigningSha = Base64.encodeToString(digest.digest(), Base64.NO_WRAP);
                     }
 
-                    String msg = "Certificate hash:\n" + packageSigningSha;
+                    String msg = packageSigningSha;
 
                     if (isAnAuthenticatorApp(pkgName)) {
                         msg += "\n\n" + getAuthenticatorAppMetadata(pkgName);
                     }
 
+                    LayoutInflater layoutInflater = (LayoutInflater) MainActivity.this.getSystemService(LAYOUT_INFLATER_SERVICE);
```

Comment:
Alternatively `LayoutInflater.from(context)` for marginally shorter line

Reply:
```suggestion
                    LayoutInflater layoutInflater = LayoutInflater.from(MainActivity.this);
```